app/features/ciip-mvp.md

```diff
@@ -0,0 +1,57 @@
+CIIP MVP (70%)
+==============
+
+Current target: December 27th.
+
+Industry can apply for CIIP
+---------------------------
+
+  - login ([authentication](./industry/authentication.feature) + [registration](./industry/registration.feature))
+  - refactored application form to enable [pre-fill of data from SWRS](./industry/dry.feature)
+  - can [view](./industry/ciip-filing.feature) previous years reports
+  - can [resume](./industry/ciip-filing.feature) in progress applications
+  - [admin data pre-filled](./industry/dry.feature) from SWRS for every reporting year
+  - ability for reporter to view flagged data and [correct it](./industry/ciip-filing.feature)
+  - can receive [email notifications](./industry/email.feature) from CIIP portal
+  - email notification could contain a link to [flagged data for correction](./industry/email.feature)
+  - [certification](./industry/ciip-filing.feature) request
+  - [legal disclaimer](./industry/registration.feature)
+  - incorporation of feedback from first user study [PENDING]
+
+System Admin
+------------
+
```

Comment:
Admin registration and authentication both need to be linked to this section

Reply:
This is a transitional document, aimed only at making it easier to reconcile these features against our poster...which didn't include admin registration and authentication. Are those now in scope for our MVP? @Maralsotoudehnia @hamzajaved

Reply:
Once answered, if they are NOT mvp then you should remove your @mvp tag on both of them :)

Reply:
ok, so this needs @Maralsotoudehnia clarification to address

Reply:
Worth a quick discussion in person with @Maralsotoudehnia

Reply:
I thought admin registration and authentication (i.e., security things) were going to be ongoing efforts that can only be fully implemented after MVP. @wenzowski am I misinterpreting? I can jump on a call before 3 today. Let me know if you want to discuss.